Cargo.toml

```diff
@@ -95,27 +93,25 @@ backend-cranelift = [
     "wasmer-clif-backend/generate-debug-information",
     "wasmer-runtime-core/generate-debug-information",
     "wasmer-runtime/cranelift",
-    "wasmer-middleware-common-tests/clif",
 ]
 backend-llvm = [
     "wasmer-llvm-backend",
     "wasmer-runtime/llvm",
-    "wasmer-middleware-common-tests/llvm",
     "wasmer-runtime-core/generate-debug-information-no-export-symbols"
 ]
 backend-singlepass = [
     "wasmer-singlepass-backend",
     "wasmer-runtime/singlepass",
-    "wasmer-middleware-common-tests/singlepass",
 ]
 wasi = ["wasmer-wasi"]
 experimental-io-devices = ["wasmer-wasi-experimental-io-devices"]
 managed = ["backend-singlepass", "wasmer-runtime-core/managed"]
 
 [[example]]
 name = "plugin"
+required-features = ["wasi"]
 crate-type = ["bin"]
 
 [[example]]
 name = "callback"
-crate-type = ["bin"]
+crate-type = ["bin"]
```

Comment:
Hmm, usually when these get removed it's due to some tool doing it.  I don't know if rustfmt affects cargo toml files (not sure what else it could be) 🤔

Reply:
Or maybe I accidentally deleted it 😆